Lib/test/test_inspect.py

```diff
@@ -24,8 +24,8 @@
 except ImportError:
     ThreadPoolExecutor = None
 
-from test.support import run_unittest, TESTFN, DirsOnSysPath, cpython_only
-from test.support import MISSING_C_DOCSTRINGS, cpython_only
+from test.support import (run_unittest, TESTFN, DirsOnSysPath, cpython_only,
+                          MISSING_C_DOCSTRINGS)
```

Comment:
This doesn't look like a relevant change to bpo-16355. Please send a separate PR for this.

Anyway, I prefer hanging indentation style:

```py
from test.support import (
    MISSING_C_DOCSTRINGS, TESTFN, DirsOnSysPath, cpython_only, run_unittest,
)
```